app/models/user.rb

```diff
@@ -1,10 +1,10 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
The user is unable to log in after creating an account. If we remove `: confirmable` then the user can log in successfully.
Kindly see the screenshot below which asks to confirm email first but this feature is not yet implemented. Please make sure the user can log in with a valid email and password.
![confirm](https://user-images.githubusercontent.com/35504149/151013756-b12766b9-1dca-4716-af21-6061104f9a21.png)

Reply:
Hi, @SajjadAhmad14. We had a requirement saying that we had to make sure that the email is verified, reference here:
![Screenshot_2022-01-25_21-50-27](https://user-images.githubusercontent.com/59100477/151057469-f20ea187-83b8-4001-bd5c-159e751cd72d.png)

If this is not correct, then what are we supposed to do with the confirmation email?

Reply:
Hi, so it's a requirement to verify email before signing in to your project. Please make sure that the user is able to sign up and sign in as well.